src/evaluation/factuality.py

```diff
@@ -142,6 +142,11 @@ def evaluate_summary(
         count_entity_labels[ANNOTATION_LABELS["Factual"]]
         + count_entity_labels[ANNOTATION_LABELS["Non-factual"]]
     )
+    # Replace count entity extrinsic with labeled if there are no unknowns
+    # We rely on human labels rather than ent["in_source"], i.e.
+    # when "Ghana" is in source, but "Ghanian" is not, we still treat "Ghanian" as intrinsic
+    if count_entity_labels[ANNOTATION_LABELS["Unknown"]] == 0:
```

Comment:
so this conditional is just to facilitate temporary results (before we've annotated everything)? Generally I think not not relying on in_source and just our annotations is a good idea

Reply:
Yupp